client.go

```diff
@@ -381,7 +384,7 @@ func (s *Client) LoopOutQuote(ctx context.Context,
 	return &LoopOutQuote{
 		SwapFee:      swapFee,
```

Comment:
`swapFee` helper variable, while `quote.PrepayAmount` is copied into the response directly.